src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -83,39 +70,17 @@ public CCDCaseApi(
 
     public List<Claim> getBySubmitterId(String submitterId, String authorisation) {
         User user = userService.getUser(authorisation);
-        return extractClaims(searchOnlyOpenCases(user, ImmutableMap.of("case.submitterId", submitterId)));
+        List<CaseDetails> result = searchAll(user, ImmutableMap.of("case.submitterId", submitterId));
+        List<CaseDetails> validCases = result.stream().filter(c -> !isCaseOnHold(c)).collect(Collectors.toList());
```

Comment:
Instead of creating two arrays I would inline it to be:

```
searchAll(user, ImmutableMap.of("case.submitterId", submitterId)).stream()
  .filter(c -> !isCaseOnHold(c))
  .collect(Collectors.toList())
```